loadbalancer/zk/logwrapper.go

```diff
@@ -0,0 +1,21 @@
+package zk
+
+import (
+	"fmt"
+	"github.com/go-kit/kit/log"
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+type wrapLogger struct {
+	log.Logger
+}
```

Comment:
No it's not a convenience function. wrapLogger wraps a go-kit logger and adds the Printf method so it can be used as the preferred logger in the underlying ZooKeeper library.